src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -60,10 +51,10 @@ import { RadioPopoverContent } from "components/molecules/RadioPopoverContent";
 import UpcomingTickets from "components/molecules/UpcomingTickets";
 import { VenuePartygoers } from "components/molecules/VenuePartygoers";
 
-import { BackButton } from "components/atoms/BackButton";
 import { ButtonNG } from "components/atoms/ButtonNG/ButtonNG";
 import { UserAvatar } from "components/atoms/UserAvatar";
-import { VolumeControl } from "components/atoms/VolumeControl";
+
+import VolumePopOverContent from "../VolumePopOverContent/VolumePopOverContent";
```

Comment:
Side note, add `index.ts` that re-exports the component like others and will shorten the import to it as well as use full path if they aren't at `./` or subdirectories of it.
```suggestion
import VolumePopOverContent from "components/molecules/VolumePopOverContent";
```